cupy/linalg/decomposition.py

```diff
@@ -82,6 +85,12 @@ def qr(a, mode='reduced'):
             ``R`` with dimensions ``(M, K)``, ``(K, N)``, where
             ``K = min(M, N)``.
```

Comment:
How about:
*The default mode is 'reduced', in which a matrix A = (M, N) is decomposed into ...*